### PR TITLE
adds include key

### PR DESCRIPTION
**Pull request by davidchase**
Let me know if you prefer me not to add helper methods or anything else should be added or removed

thanks

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -53,13 +63,17 @@ NYC.prototype._createInstrumenter = function () {
 NYC.prototype.addFile = function (filename, returnImmediately) {
   var instrument = true
   var relFile = path.relative(this.cwd, filename)
+  var include
 
-  // only instrument a file if it's not on the exclude list.
   for (var i = 0, exclude; (exclude = this.exclude[i]) !== undefined; i++) {
+    include = this.include[i]
```
Comment (Lalem001) [inline index.js]: I see a potential problem here. If the exclude list is shorter than the include list, it will not get through all the includes.

Comment (bcoe) [inline index.js]: @Lalem001 good find, perhaps you could add a couple tests @davidchase -- was going to bug you about this already.

I haven't forgotten about this pull request, I've been doing a big overhaul of nyc that makes it work properly with ES6 :) I hope to have a few releases of the project out this weekend.

**Comment by bcoe** [comment]
@davidchase hey, mind updating with tests? Perhaps you can coordinate your work with the improvements @Lalem001 is working on in:

https://github.com/bcoe/nyc/pull/60

**Comment by davidchase** [comment]
sorry i have been pretty busy and didnt have a chance to update this PR... maybe its beneficial to close this and have @Lalem001 create one or something of the sort... sorry

**Comment by bcoe** [comment]
@davidchase addressed in https://github.com/bcoe/nyc/pull/84

**Pull request closed without merging**